launch/test/launch/test_launch_service.py

```diff
@@ -14,21 +14,14 @@
 
 """Tests for the LaunchService class."""
 
+import asyncio
+import osrf_pycommon
```

Comment:
`osrf_pycommon` isn't a system library, so it belongs in a separate block.

Reply:
True that, and `asyncio` shouldn't be there anymore. Fixed.